autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -144,7 +146,7 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
                 KNN: See sklearn documentation: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html
                     Note: Hyperparameter tuning is disabled for this model.
                     Note: 'weights' parameter will be overriden. Both 'distance' and 'uniform' are used automatically, training two models.
-
+        
```

Comment:
Very minor but I notice this often with your PR's. In my PyCharm IDE it automatically removes this whitespace and thus in my commits it is being removed. So when we work on the same code this line gets changed back and forth. I'm fine with either way but I'm curious your thoughts on it.

Seems people are fairly divided on what is best: https://stackoverflow.com/questions/2727988/python-indentation-in-empty-lines